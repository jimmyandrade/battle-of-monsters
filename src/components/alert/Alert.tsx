import type { FC, HTMLAttributes } from 'react'
import { AlertRoot } from './Alert.styles'

export const Alert: FC<HTMLAttributes<HTMLDivElement>> = (
  {
    'aria-live': ariaLive = 'polite',
    className,
    ...props
  }) => (
  <AlertRoot aria-live={ariaLive} className={className} {...props} />
)
