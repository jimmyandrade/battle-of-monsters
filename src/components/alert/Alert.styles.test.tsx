import { render } from '@testing-library/react'
import { AlertRoot } from './Alert.styles'

describe('Alert styles', () => {
  describe('Root component', () => {
    it('should match snapshot', () => {
      const { container } = render(<AlertRoot />)
      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
