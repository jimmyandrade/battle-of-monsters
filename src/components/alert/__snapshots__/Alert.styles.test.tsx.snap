// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert styles Root component should match snapshot 1`] = `
.c0 {
  background-color: #E1F8FF;
  border: 1px solid var(--active-border-color);
  border-radius: 4px;
  box-shadow: var(--box-shadow);
  font-size: 22px;
  min-height: 61px;
  padding: 17px 28px 18px 28px;
}

<div
  class="c0"
/>
`;
